src/components/App/App.js

```diff
@@ -1,46 +1,23 @@
 import React, { Component } from 'react';
-import Movie from '../Movies/Movies';
+import Movies from '../Movies/Movies';
+import movieData from '../../movieData';
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: [
-        {
-          "id": 694919,
-          "poster_path": "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg",
-          "backdrop_path": "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg",
-          "title": "Money Plane",
-          "average_rating": 6.666666666666667,
-          "release_date": "2020-09-29"
-        },
-        {
-          "id": 337401,
-          "poster_path": "https://image.tmdb.org/t/p/original//aKx1ARwG55zZ0GpRvU2WrGrCG9o.jpg",
-          "backdrop_path": "https://image.tmdb.org/t/p/original//zzWGRw277MNoCs3zhyG3YmYQsXv.jpg",
-          "title": "Mulan",
-          "average_rating": 4.909090909090909,
-          "release_date": "2020-09-04"
-        },
-        {
-          "id": 718444,
-          "poster_path": "https://image.tmdb.org/t/p/original//uOw5JD8IlD546feZ6oxbIjvN66P.jpg",
-          "backdrop_path": "https://image.tmdb.org/t/p/original//x4UkhIQuHIJyeeOTdcbZ3t3gBSa.jpg",
-          "title": "Rogue",
-          "average_rating": 5.428571428571429,
-          "release_date": "2020-08-20"
-        },
-      ]
+      movies: movieData.movies
     }
   }
  
   render() {
```

Comment:
Nice and clean basic setup of the movies, ask about whether a separate Nav component is needed.